docs/admin/kubeadm.md

```diff
@@ -104,17 +105,16 @@ and `--external-etcd-keyfile` flags.
 
 - `--pod-network-cidr`
 
-By default, `kubeadm init` does not set node CIDR's for pods and allows you to
-bring your own networking configuration through a CNI compatible network
-controller addon such as [Weave Net](https://github.com/weaveworks/weave-kube),
-[Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm)
-or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm).
-If you are using a compatible cloud provider or flannel, you can specify a
-subnet to use for each pod on the cluster with the `--pod-network-cidr` flag.
-This should be a minimum of a /16 so that kubeadm is able to assign /24 subnets
-to each node in the cluster.
+For certain networking solutions the Kubernetes master can also play a role in
+allocating network ranges (CIDRs) to each node. This includes many cloud providers
+and flannel. You can specify a subnet range that will be broken down and handed out
+to each node with the `--pod-network-cidr` flag. This should be a minimum of a /16 so
+controller-manager is able to assign /24 subnets to each node in the cluster.
+If you are using flannel with [this manifest](https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml)
+you should use `--pod-network-cidr=10.244.0.0/16`. Most CNI based networking solutions
+do not require this flag. This includes Weave, Calico and Canal.
```

Comment:
Sorry to keep chiming in on this passage.. I really don't think we want to be enumerating networking solutions here (and most other places)!

It becomes way to easy to get out of date, and is misleading to users who now think that Weave, Calico + Canal are privileged in some way.

e.g Why isn't Romana in this list - that's also a working, documented addon :)  Will we add every networking addon that doesn't need this flag here? I think not.

On a side note, can we _please_ alphabetize or ordering lists like this in some standard way when we do actually need them?